doc/nrf/applications.rst

```diff
@@ -17,7 +17,7 @@ If you want to list applications available for one or more specific boards, `use
    :glob:
    :caption: Subpages:
 
-   ../../applications/asset_tracker_v2/README
+   ../../applications/asset_tracker/README
```

Comment:
As per comment above, we should not have a README for a non-existent app. This should be a redirect.